cmake/protobuf-targets.cmake

```diff
@@ -0,0 +1,17 @@
+# Library aliases
+foreach(_library

```

Comment:
ALIASES only work when the original targets are also defined. If you do this, when a downstream project's find_package is pointed at the build directory from a project that doesn't include all of protobuf as a sub_directory it will fail terribly. CMP0024 disallows including the result of an export command, but that only happens because we're calling find_package(protobuf...) from *within* protobuf. That function was never meant to do that.